### PR TITLE
feat: article k8s local

### PR DESCRIPTION
**Pull request by rguichard**
Must be merged after https://github.com/particuleio/navigator-hugo/pull/13

### CODE REVIEW

File: content/blog/kubernetes-local.fr.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+Author: Romain Guichard
+image: images/thumbnails/kubernetes.png
+imgSocialNetwork: images/og/kubernetes-local.png
+lang: fr
+---
+
+Comment travailler avec Kubernetes ? De quoi a t-on besoin ? Ce sont des
+questions qui reviennent régulièrement et pour cause, lorsqu'on étudie
+Kubernetes et son fonctionnement, on tombe rapidement sur des configurations
+matérielles ou virtuelle relativement complexes. Le cluster etcd peut être
```
Comment (tbobm) [inline content/blog/kubernetes-local.fr.md]: An `s` seems to be missing in `virtuelle`

File: content/blog/kubernetes-local.fr.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+installé sur les mêmes nodes hébergeant l'api-server et les autres éléments du
+control plane. Le control plane peut être constitué de un ou plusieurs nodes en
+fonction du niveau de résilience souhaité. Pareil pour etcd. De multiples
+configurations sont possibles et chacune à ses avantages et ses inconvénients.
```
Comment (tbobm) [inline content/blog/kubernetes-local.fr.md]: Minor typo in `chacune a ses`

File: content/blog/kubernetes-local.fr.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+Une fois que cette configuration est choisie, la seconde question que l'on se
+pose est bien souvent "où ?". En effet, l'endroit où sera déployé Kubernetes
+est un choix important puisque certaines fonctions (Service loadbalancer,
+Storage Class, etc) ne seront peut être pas disponibles.
```
Comment (tbobm) [inline content/blog/kubernetes-local.fr.md]: Minor typo in `peut-être` (missing `-`)

File: content/blog/kubernetes-local.fr.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+un nombre important de projet tiers comme la solution pour bootstraper un
+cluster Kubernetes. Mais c'est là sa limite, il ne fait que ça. En soit pour
+démarrer un control plane Kubernetes, il suffit de :
+
+```
+$ kubeadm init
+```
+
+Mais pour bénéficier en sortie d'un control plane fonctionnel, cela
+présupposeque plusieurs choses soient installées, en amont ou en aval :
```
Comment (tbobm) [inline content/blog/kubernetes-local.fr.md]: Minor typo in `présupposeque` (missing whitespace between `présuppose` and `que`)

File: content/blog/kubernetes-local.fr.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+k3s se compose d'un binaire d'une cinquantaine de Mo. Il suffit de le lancer pour
+bénéficier d'un cluster Kubernetes up and running.
+
+```
+$ k3s kubectl get node
+NAME           STATUS   ROLES                  AGE    VERSION
+rguichard-x1   Ready    control-plane,master   345d   v1.20.0+k3s2
+```
+
+Est inclu :
```
Comment (tbobm) [inline content/blog/kubernetes-local.fr.md]: Maybe replace `Est inclu` with `Sont inclus` ?

File: content/blog/kubernetes-local.fr.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+k3d-third-agent-1    Ready    <none>        11s   v1.19.4+k3s1
+k3d-third-agent-2    Ready    <none>        10s   v1.19.4+k3s1
+k3d-third-server-0   Ready    etcd,master   50s   v1.19.4+k3s1
+k3d-third-server-1   Ready    etcd,master   36s   v1.19.4+k3s1
+k3d-third-server-2   Ready    etcd,master   17s   v1.19.4+k3s1
+```
+
+## Et les autres ?
+
+D'autres solutions existent évidemment, nous n'avons choisi de ne pas les
```
Comment (tbobm) [inline content/blog/kubernetes-local.fr.md]: Maybe remove the `n'` from `nous [n']avons choisi` ?

**Review by tbobm — COMMENTED** [review]
I've noticed a couple of minor typos / wording issues

**Comment by rguichard** [comment]
Fixes apply.

File: content/blog/kubernetes-local.fr.md [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+![kind](https://d33wubrfki0l68.cloudfront.net/d0c94836ab5b896f29728f3c4798054539303799/9f948/logo/logo.png#logosize)
+
+[Kind](https://kind.sigs.k8s.io/) est un outil de déploiement de Kubernetes
+géré par le Sig Testing et hébergé sous
+[kubernetes-sigs](https://github.com/kubernetes-sigs/kind/). Kind permet de
+déployer des clusters Kubernetes en se servant de conteneurs Docker comme des
+nodes Kubernetes. Chaque node utilise kubeadm afin de bootstraper le cluster ou
+de rejoindre un cluster existant.
+
+Malgré le fait que [Docker est désormais deprecated par
```
Comment (ArchiFleKs) [inline content/blog/kubernetes-local.fr.md]: Weird phrasing, remove "Malgré le fait que"

**Review by ArchiFleKs — CHANGES_REQUESTED** [review]

**Review by ArchiFleKs — APPROVED** [review]

**Pull request merged**